events/templates/events/event_details.html

```diff
@@ -5,12 +5,11 @@
 </tr><tr>
 	<td><b>{% trans "Place:" %}</b></td><td>{{ event.place }}</td>
 <tr>
-	<td><b>{% trans "Host:" %}</b></td><td>{{ event.created_by }}</td>
+	<td><b>{{ theme.text_labels.host }}</b></td><td>{{ event.created_by }}</td>
```

Comment:
Since the themed text doesn't end in a colon, please add it after the text insertion here.